front/src/modules/pathfinding/components/Itinerary/DisplayViasV2.tsx

```diff
@@ -19,6 +92,7 @@ const DisplayViasV2 = ({ zoomToFeaturePoint }: DisplayViasV2Props) => {
   const dispatch = useAppDispatch();
   const vias = useSelector(getViasV2());
   const { moveVia, deleteViaV2 } = useOsrdConfActions();
+  const [focusedViaId, setFocusedViaId] = useState<string>();
```

Comment:
You can move this in StopDurationSelector component
```suggestion
```

Reply:
I want to keep it in DisplayVias, otherwise we can focus several inputs at the same time
![image](https://github.com/OpenRailAssociation/osrd/assets/45000526/9adc8737-86fe-4add-939f-69ca3f026c8c)